shared/featureconfig/flags.go

```diff
@@ -625,5 +625,4 @@ var E2EBeaconChainFlags = []string{
 	"--check-head-state",
 	"--attestation-aggregation-strategy=max_cover",
 	"--dev",
-	"--enable-finalized-deposits-cache",
```

Comment:
This will be fixed by #6588.

```suggestion
   // "--enable-finalized-deposits-cache", // TODO(6588): enable in e2e
```